phpstan.neon

```diff
@@ -0,0 +1,6 @@
+parameters:
+    autoload_files:
+        - tests/bootstrap.php
+    ignoreErrors:
+        - '#Call to an undefined method Cake\\Datasource\\RepositoryInterface::aliasField\(\)#'
```

Comment:
This should also have an annotation that can be taught, right?

Reply:
Not that I know of. The code already has a `method_exists()` check before trying to use that method so there's no actual error. I don't know how to refactor the code to make phpstan aware of that (if it can be even made to understand).

Reply:
We can also try to remove the ignors later of course.
Would have just been nice to not have to.